From bb2eef7b83a1114dadc01d8bd9bd0dabc001e70a Mon Sep 17 00:00:00 2001
From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Mon, 12 Feb 2018 21:37:52 +0100
Subject: [PATCH 043/192] ungoogled-chromium: Disable Gaia

Disables Gaia code.
Somehow it is still being activated even without being signed-in.

See also: https://github.com/Eloston/ungoogled-chromium/issues/104

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 google_apis/gaia/gaia_auth_fetcher.cc | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/google_apis/gaia/gaia_auth_fetcher.cc b/google_apis/gaia/gaia_auth_fetcher.cc
index 7d789c40d3847..c87a307d8a2da 100644
--- a/google_apis/gaia/gaia_auth_fetcher.cc
+++ b/google_apis/gaia/gaia_auth_fetcher.cc
@@ -41,7 +41,7 @@
 
 namespace {
 
-const size_t kMaxMessageSize = 1024 * 1024;  // 1MB
+//const size_t kMaxMessageSize = 1024 * 1024;  // 1MB
 
 constexpr char kBadAuthenticationError[] = "BadAuthentication";
 constexpr char kBadAuthenticationShortError[] = "badauth";
@@ -272,6 +272,7 @@ void GaiaAuthFetcher::CreateAndStartGaiaFetcher(
     const net::NetworkTrafficAnnotationTag& traffic_annotation) {
   DCHECK(!fetch_pending_) << "Tried to fetch two things at once!";
 
+/*
   auto resource_request = std::make_unique<network::ResourceRequest>();
   resource_request->url = gaia_gurl;
   original_url_ = gaia_gurl;
@@ -326,6 +327,7 @@ void GaiaAuthFetcher::CreateAndStartGaiaFetcher(
       base::BindOnce(&GaiaAuthFetcher::OnURLLoadComplete,
                      base::Unretained(this)),
       kMaxMessageSize);
+*/
 }
 
 // static
-- 
2.34.1

